src/pickers/ClaimOfficerPicker.js

```diff
@@ -21,47 +23,50 @@ const ClaimOfficerPicker = (props) => {
   const modulesManager = useModulesManager();
   const { formatMessage } = useTranslations("claim", modulesManager);
   const [variables, setVariables] = useState({});
+  const dispatch = useDispatch();
 
-  const { isLoading, data, error } = useGraphqlQuery(
-    `
-    query ClaimOfficerPicker ($search: String) {
-      claimOfficers(search: $search, first: 20) {
-        edges {
-          node {
-            id
-            uuid
-            code
-            lastName
-            otherNames
-            ${extraFragment ?? ""}
-          }
-        }
-      }
-    }
-  `,
-    variables,
-    { skip: true },
-  );
+  const isLoading = useSelector((state) => state.claim.claimOfficers.isFetching);
```

Comment:
Can you confirm that `state.claim.claimOfficers` and other values fetched with useSelector hook are always available? 
If there's use case in which claimOfficers variable is undefined then we'll face `TypeError: Cannot read property..` error. Maybe we should add check to fetching functions, example: `useSelector((state) => state.claim.claimOfficers? state.claim.claimOfficers.isFetching: null);`.